rocketpy/AeroSurface.py

```diff
@@ -1892,53 +1885,113 @@ def allInfo(self):
         return None
 
 
-class RailButtons:
+class RailButtons(AeroSurface):
     """Class that defines a generic rail button.
 
     Attributes
     ----------
-    RailButtons.upper_button_position : int, float
-        Position of the upper rail button in meters. The upper button is the one
-        closest to the nose cone and furthest from the tail. The coordinate system
-        used is the same as the Rocket that the buttons will be a part of.
-    RailButtons.lower_button_position : int, float
-        Position of the lower rail button in meters. The lower button is the one
-        closest to the nose cone and furthest from the tail. The coordinate system
-        used is the same as the Rocket that the buttons will be a part of.
+    RailButtons.buttons_distance : int, float
+        Distance between the highest and the lowest rail button in meters.
     RailButtons.angular_position : int, float
         Angular position of the rail buttons in degrees measured
         as the rotation around the symmetry axis of the rocket
         relative to one of the other principal axis.
     """
 
-    def __init__(self, upper_button_position, lower_button_position, angular_position):
+    def __init__(self, buttons_distance, angular_position=45, name="Rail Buttons"):
         """Initializes RailButtons Class.
 
         Parameters
         ----------
-        upper_button_position : int, float
-            Position of the upper rail button in meters. The upper button is the one
-            closest to the nose cone and furthest from the tail. The coordinate system
-            used is the same as the Rocket that the buttons will be a part of.
-        lower_button_position : int, float
-            Position of the lower rail button in meters. The lower button is the one
-            closest to the nose cone and furthest from the tail. The coordinate system
-            used is the same as the Rocket that the buttons will be a part of.
+        buttons_distance : int, float
+            Distance between the first and the last rail button in meters.
         angular_position : int, float, optional
             Angular position of the rail buttons in degrees measured
             as the rotation around the symmetry axis of the rocket
             relative to one of the other principal axis.
+        name : string, optional
+            Name of the rail buttons. Defaul is "Rail Buttons".
 
         Returns
         -------
         None
 
         """
-        self.upper_button_position = upper_button_position
-        self.lower_button_position = lower_button_position
+        self.buttons_distance = buttons_distance
```

Comment:
I remember the distance vs positions PR discussions. Since the class already includes a pair of railbuttons and their distance is working here more like an mathematical value than a core attribute, I will not object too much.

Even thought it would increase code parallelism, it might be a bit "too much" to define another coordinate system here.